src/core/socket-bind.c

```diff
@@ -25,12 +25,10 @@ DEFINE_TRIVIAL_CLEANUP_FUNC(struct socket_bind_bpf *, socket_bind_bpf_free);
 static int update_rules_map(
                 int map_fd, CGroupSocketBindItem *head) {
         CGroupSocketBindItem *item;
-        uint32_t i = 0;
 
         assert(map_fd >= 0);
 
         LIST_FOREACH(socket_bind_items, item, head) {
-                const uint32_t key = i++;
```

Comment:
This won't compile: `key` is required to add an element to the map:
```
                 if (bpf_map_update_elem(map_fd, &key, &val, BPF_ANY) != 0)
```

Reply:
uh, gah, in my test env libbpf wasn't installed. let me roll this again. sorry for the confusoin.